webrender/src/frame.rs

```diff
@@ -282,6 +282,19 @@ impl Frame {
             let mut traversal = DisplayListTraversal::new_skipping_first(display_list);
             let reference_frame_id = self.scroll_tree.root_reference_frame_id();
             let topmost_scroll_layer_id = self.scroll_tree.topmost_scroll_layer_id();
+
+
+            let viewport_rect = LayerRect::new(LayerPoint::zero(), root_pipeline.viewport_size);
+            let clip = ClipRegion::simple(&viewport_rect);
+            context.builder.push_scroll_layer(reference_frame_id,
+                                              &clip,
+                                              &LayerPoint::zero(),
+                                              &root_pipeline.viewport_size);
+            context.builder.push_scroll_layer(topmost_scroll_layer_id,
```

Comment:
ScrollTree::establish_root should always create two separate layers for these, so this shouldn't happen. I can add an assertion here though.